### ISSUE TITLE
Repeated implantation failure

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6810]
**Preferred term label:**
Repeated implantation failure

**Synonyms**
Recurrent implantation failure

**Definition (free text, please give PubMed ID)**
Repeated implantation failure (RIF) is determined when embryos of good quality fail to implant following several in vitro fertilization (IVF) treatment cycles.
PMID:25473625
PMID:22976427

Comment
Successful embryo implantation is a process which requires both a synchronous development and interaction between hatched blastocyst and endometrium. From the clinical point of view, implantation is considered to be successful when gestational sac is diagnosed by ultrasound. 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619176 
PMID:31704776  2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
